common/utils/yaml.go

```diff
@@ -0,0 +1,52 @@
+package utils
+
+import "gopkg.in/yaml.v3"
+
+func MergeYAMLString(oldValues string, newValues string) (string, error) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Probably opinionated
```suggestion
func MergeYAML(oldValuesYamlContent string, newValuesYamlContent string) (string, error) {
```